images/egress-router-cni.yml

```diff
@@ -4,11 +4,9 @@ container_yaml:
     - module: github.com/openshift/egress-router-cni
 content:
   source:
-    dockerfile: Dockerfile.rhel8
-    git:
-      branch:
-        target: release-{MAJOR}.{MINOR}
-      url: git@github.com:openshift-priv/egress-router-cni.git
+    alias: egress-router-cni-alt
```

Comment:
Doozer auto registers alias for all sources, but I found this trick doesn't always work because Doozer may resolve the `-alt` images before non`-alt` images. So I will go back to duplicate the source definitions :(